plugins/generator-1.19.4/forge-1.19.4/procedures/team_add_entity.java.ftl

```diff
@@ -0,0 +1,6 @@
+{
+	Entity _entity${cbi} = ${input$entity};

```

Comment:
${cbi} is only needed for ouput / inline statements. Here you can use eg _entityTeamAdd